pubtools/_quay/command_executor.py

```diff
@@ -245,3 +258,163 @@ def _run_cmd(self, cmd, err_msg=None, tolerate_err=False, stdin=None):
                 raise RuntimeError(err_msg)
 
         return out_text, err_text
+
+
+class ContainerExecutor(Executor):
+    """Run commands in a Docker container."""
+
+    def __init__(
+        self,
+        image,
+        base_url="unix://var/run/docker.sock",
+        timeout=None,
+        verify_tls=False,
+        cert_path=None,
+    ):
+        """
+        Initialize.
+
+        Args:
+            image (str):
+                Path to an image which will be used for performing skopeo operations. Must be
+                downloadable by Docker.
+            base_url (str):
+                Base URL of the Docker client.
+            timeout (int):
+                Default timeout for API calls, in seconds.
+            verify_tls (bool):
+                Whether to use TLS verification.
+            cert_path (str|None):
+                Custom path to TLS cettificates.
```

Comment:
`s/tt/rt/`

also it could be useful to mention that `~/.docker/` is used by default. Maybe even make it the default arg value instead of `None` and simply `os.path.expanduser(cert_path)`?

Reply:
Fixed typo and added information about default location. This part is mostly copy-pasted from pub so I'd prefer to leave it as is, if that's okay.